server/mcp_server_vedb_mysql/README.md

```diff
@@ -14,37 +14,109 @@
 
 ## Tools
 
-### 1. `create_vedb_mysql_instance`
-- **Description**: Creates a new instance.
-- **Trigger example**: `"Create a new vedbm instance using the parameters of my other instance as reference, and tell me the MySQL connection string."`
 
-### 2. `list_vedb_mysql_instances`
-- **Description**: Retrieves the list of a user’s veDB MySQL instances, including each instance ID and basic information.
-- **Trigger example**: `"List my vedbm instances."`
-
-### 3. `describe_vedb_mysql_detail`
-- **Description**: Gets detailed information about a specified instance.
-- **Trigger example**: `"Show the details of instance ID vedbm-hylviolixpvu."`
-
-### 4. `list_vedb_mysql_instance_databases`
-- **Description**: Retrieves the list of databases within a specified instance, including permission information.
-- **Trigger example**: `"Show the database list in instance vedbm-hylviolixpvu."`
-
-### 5. `list_vedb_mysql_instance_accounts`
-- **Description**: Retrieves account information within a specified instance, including permission details.
-- **Trigger example**: `"Show the account list in instance vedbm-hylviolixpvu."`
-
-### 6. `modify_vedb_mysql_instance_alias`
-- **Description**: Updates the alias of a specified instance.
-- **Trigger example**: `"Change the alias of instance vedbm-hylviolixpvu to Production Database."`
-
-### 7. `create_vedb_mysql_allowlist`
-- **Description**: Creates a veDB MySQL network allowlist (whitelist).
-- **Trigger example**: `"Create a veDB MySQL network allowlist only for my ECS instances."`
-
-### 8. `bind_allowlist_to_vedb_mysql_instances`
-- **Description**: Binds a veDB MySQL network allowlist to one or more instances.
-- **Trigger example**: `"Bind the allowlist created in the previous step to my veDB MySQL instance."`
+### Instance Management
```

Comment:
focus on mostly used tool(openapi), dont' support all openapi in mcp. make sure your use case with prompt is working as expected.